### DISCUSSION TITLE
Press ( Tab key ) add a box on the option ...

### DISCUSSION THREAD

**Discussion by ioc2e3** [goastian/midori-desktop#48] [Ideas]
Press ( Tab key ) add a box on the option ...

Users can know what options have been pressed ...

Firefox Browser is like this ...
![rrr](https://github.com/goastian/midori-desktop/assets/152480314/3fa18058-9b92-467d-86e8-c08fd4464571)

Can Midori Browser do this?

.

**Comment by Ponchale** [comment]
We're finalizing the final details before launch, but we promise to fix this
